test/tests/meshgenerators/boundary_deletion_generator/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  [./boundary_deletion_test]
+    type = JSONDiff
+    input = 'boundary_deletion.i'
+    jsondiff = 'boundary_deletion_out.json'
+    mesh_mode = 'REPLICATED'
```

Comment:
Remove this requirement. Should work fine with distributed mesh. Will also likely need to add `allow_renumbering = false` to the input.